app/components/header.module.css

```
@@ -184,3 +185,13 @@
         margin-right: var(--space-2xs);
     }
 }
+
+.sudo > button {
```

```suggestion
.sudo-menu-item {
```

or am I missing something? 😅

---

Only the 4123 refactors that occurred before I pushed the branch.